### PR TITLE
update ops's unittest data type from float32 to float64 and shape over 100 

### LINKED ISSUE #22542: 升级elementwise_pow、elementwise_max、elementwise_min、scale、sqrt等op单测精度、shape
1. 将elementwise_pow、elementwise_max、elementwise_min、scale、sqrt等op单测精度 由float32 升级至float64。
 * 其中sqrt发现目前develop分支已经是float64，并且将sqrt的单测从 https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/white_list/op_accuracy_white_list.py 移除，单独跑sqrt的单测没有报错（前后向误差满足要求），故从op_accuracy_white_list.py移除sqrt。
 * 其中elementwise_pow目前develop分支已经是float64，但前后向误差不满足阈值要求。
image

2.将elementwise_pow、elementwise_max、elementwise_min等op单测输入数据 shape升级，大于等于100。

### PR DESCRIPTION
**Pull request by Steffy-zxf**
update ops's unittest of elementwise_pow, elementwise_max, elementwis…
1. update elementwise_pow, elementwise_max and scale's unitests with input data type (float32 -> float64)
2. fix bug that the elementwise_pow doesn't meet threshold requirements with tackling float64 data
3. remove sqrt from op_accuracy_white_list.py
4. update the unittests of elementwise_pow, elementwise_max and elementwise_min ops that their input data shape over 100
5. test=develop
6. fix https://github.com/PaddlePaddle/Paddle/issues/22542

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_elementwise_pow_op.py [hunk 1]
```diff
@@ -49,12 +49,14 @@ class TestElementwisePowOp_big_shape_2(TestElementwisePowOp):
     def setUp(self):
         self.op_type = "elementwise_pow"
         self.inputs = {
-            'X': np.random.uniform(0.1, 1, [10, 10]).astype("float64"),
-            'Y': np.random.uniform(0.1, 1, [10, 10]).astype("float64") * 20
+            'X': np.random.uniform(1, 2, [10, 10]).astype("float64"),
+            'Y': np.random.uniform(0.01, 0.1, [10, 10]).astype("float64") * 20
```
Comment (zhupengyang) [inline python/paddle/fluid/tests/unittests/test_elementwise_pow_op.py]: 'Y': np.random.uniform(0.2, 2, [10, 10]).astype("float64")

Comment (Steffy-zxf) [reply python/paddle/fluid/tests/unittests/test_elementwise_pow_op.py]: done

**Review by juncaipeng — DISMISSED** [review]
LGTM for upgrading the precision from fp32 to fp64.

**Review by zhupengyang — APPROVED** [review]
LGTM for shape

**Review by hong19860320 — APPROVED** [review]
LGTM

**Review by luotao1 — APPROVED** [review]

**Pull request merged**